src/main/js/html.js

```diff
@@ -641,6 +647,10 @@
 
                     /* style: top right bottom left*/
 
+                    if (attr === undefined || attr === null) {
+                      attr = [0, 0, 0, 0];
+                    }
+
```

Comment:
Under what conditions would this ever be permitted? Are you thinking `tts:padding=""`?